python-sdk/src/forta_agent/finding.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, dict):
         self.type = dict['type']
         self.metadata = dict.get('metadata')
         self.addresses = dict.get('addresses')
-        self.labels = list(map(lambda t: Label(t), dict.get('labels', [])))
+        self.labels = list(map(lambda l: l if isinstance(l, Label) else Label(l), dict.get('labels', [])))
```

Comment:
[Question] Is there a unit test pattern for field like labels to make sure they serialize correctly? Maybe we could start using types b/c python supports types now?

Reply:
+1 on using types in Python. i think we should also bump the minimum Python version to 3.9 (since 3.6 has reached end of life). this can be unit-tested but im punting for now